sklearn/preprocessing/_data.py

```diff
@@ -1130,6 +1130,10 @@ class RobustScaler(TransformerMixin, BaseEstimator):
 
         .. versionadded:: 0.18
 
+    unit_variance : boolean, False by default
+        If True, scale data so that normally distributed features have a
+        variance of 1.
+
```

Comment:
this needs a `.. versionadded` and needs to be moved to be the last argument. We're still accepting positional args in the next release.

Reply:
Could we also expand on this a bit and explain what happens if the feature is not normally distributed? Since this is the only place we're having the explanation of this parameter (no user guide), it'd be nice to elaborate a bit.

Reply:
Hmm in general if `norm.ppf(quantile_range[1], 0, 1) - norm.ppf(quantile_range[0], 0, 1)`` is greater than 1, the output data will be scaled down (decrease variance) and if less than 1 it will be scaled up. Would this be adequate..?